tools/experiments/idle.py

```diff
@@ -92,22 +105,28 @@ def kernel_loop(self, loop_index: TInt64):
         self.core.break_realtime()
 
         if differential_mode:
-            for kk in range(len(self.repump_aoms)):
+            for kk in range(len(self.repump_aoms_dds)):
                 # if the repump AOM is off, don't turn on and off the AOM.
                 # the repump AOM stays off for both differential high and low counting periods.
-                if self.repump_aom_states[kk] > 0.:
-                    self.repump_aoms[kk].sw.off()
+                if self.repump_aom_dds_states[kk] > 0.0:
+                    self.repump_aoms_dds[kk].sw.off()
+            for kk in range(len(self.repump_aoms_ttl)):
+                self.repump_aoms_ttl[kk].off()
             t_count = self.pmt_counter.gate_rising_mu(interval_mu)
 
             at_mu(t_count + self.rtio_cycle_mu)
-            for kk in range(len(self.repump_aoms)):
-                if self.repump_aom_states[kk] > 0.:
-                    self.repump_aoms[kk].sw.on()
+            for kk in range(len(self.repump_aoms_dds)):
+                if self.repump_aom_dds_states[kk] > 0.0:
+                    self.repump_aoms_dds[kk].sw.on()
+            for kk in range(len(self.repump_aoms_ttl)):
+                self.repump_aoms_ttl[kk].on()
             t_count = self.pmt_counter.gate_rising_mu(interval_mu)
         else:
             t_count = self.pmt_counter.gate_rising_mu(interval_mu)
 
-        twenty_ms_mu = 20 * ms  # 20 ms time slack to prevent slowing down PMT acquisition.
+        twenty_ms_mu = (
+            20 * ms
```

Comment:
Maybe the variable should be named `slack_time_ms`

Reply:
delay_ or wait_time_ms?